openmetadata-docs/content/openmetadata/connectors/database/oracle/airflow.md

```diff
@@ -140,7 +140,9 @@ workflowConfig:
 - **username**: Specify the User to connect to Oracle. It should have enough privileges to read all the metadata.
 - **password**: Password to connect to Oracle.
 - **hostPort**: Enter the fully qualified hostname and port number for your Oracle deployment in the Host and Port field.
-- **oracleServiceName**: The Oracle Service name is the TNS alias that you give when you remotely connect to your database and this Service name is recorded in tnsnames.
+- **oracleConnectionType** :
+  - **oracleServiceName**: The Oracle Service name is the TNS alias that you give when you remotely connect to your database and this Service name is recorded in tnsnames.
+  - **databaseSchema**: Enter the name of the database schema available in Oracle that you want to connect with, and make sure that the user has enough access to use resources available in this database.
```

Comment:
```suggestion
  - **databaseSchema**: The name of the database schema available in Oracle that you want to connect with, and make sure that the user has enough access to use resources available in this database.
```

Reply:
Do we have somewhere documented permissions needed or how to give access to a user?